cocktails9/cocktails9/SplashView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SplashScreenView.swift
+//  cocktails9
+//
+//  Created by Stasa Zujkovic on 27.11.24..
+//
+
+import SwiftUI
+
+struct SplashView: View {
+    
+    @Binding var isActive: Bool
```

Comment:
It is this view parent's job to know isActive state, splash should just do its magic when loaded. With that in mind, move isActive var and "onAppear" part to parent view.